docs/release-notes/index.rst

```diff
@@ -0,0 +1,257 @@
+.. _release-notes:
+
+Release notes
+=============
+
+This page lists past release notes for Rockcraft, summarising new features, bug
+fixes and backwards-incompatible changes in each version. It also contains the
+release and support policies for Rockcraft.
+
+
+Current releases
+----------------
+
+
+Rockcraft 1
+~~~~~~~~~~~
+
+- :ref:`release-1.6.0`.
```

Comment:
The intention here is to link to the release note pages, not the changelog. I think for the changelog we should instead link it on the release note pages themselves – the changelog is already an indexed page, so it's fairly discoverable as-is.

Reply:
ah I think I'm missing something here then - I thought these release notes were replacing the existing changelog altogether. What is the difference between the two then?
For example, looking at the changelog for [1.6.0](https://github.com/canonical/rockcraft/blob/main/docs/reference/changelog.rst) - it's a different format but it's already structured in an end-user language. Aren't the two documents redundant?

Reply:
The release notes are a new requirement of documentation sets. There's no rule against also having a changelog. When we originally discussed this I saw value in keeping the changelog for developers, but if at a later point we want to rethink the idea, I'm fine with that.

The release notes are intended to be consumer- and user-friendly copy that occupies several roles, among them _announcing_ releases more than just reporting them, highlighting developments and our intentions, and communicating to a broader audience (such as C-level, gov workers).

The two documents can exist in parallel, with the changelog covering every change at any level of minutiae (if you'd like), while with release notes we can be more selective and cover the most significant work and take the time to explain the background, motivation, use-cases, and other bits of _why_ in our progress.

Reply:
ah I see. I have a draft of release notes for 1.7.0 that I made with the notion that it was replacing the changelog, so we can use that upcoming PR to discuss which entries "deserve" to be in the release notes and which notes can go on the changelog.

Reply:
Sounds good to me! :)